app/src/main/res/layout/item_contact.xml

```diff
@@ -4,26 +4,54 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    xmlns:fresco="http://schemas.android.com/apk/res-auto"
     android:background="?android:attr/selectableItemBackground"
     android:paddingStart="@dimen/screen_edge_left_and_right_padding"
     android:paddingTop="@dimen/chat_item_top_and_bottom_padding"
     android:paddingEnd="@dimen/screen_edge_left_and_right_padding">
 
+    <RelativeLayout
+        android:id="@+id/rl_image_avatar"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:padding="10dp">
+
+    <com.facebook.drawee.view.SimpleDraweeView
```

Comment:
Should we have a simple default avatar?  Or, will there always be some kind of avatar to discover?  It seems like the contacts that are not synced will need a default avatar?

Reply:
ok